sites/docs/src/content/docs/usage/configuration/modify_tools.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Tool arguments
+subtitle: Configure tool containers and arguments
+shortTitle: Tool arguments
+weight: 3
+---
+
+## Modifying tools
+
+Each tool in an nf-core pipeline come preconfigured with a set arguments for an average user.
+The arguments are a great place to start and have been tested as a part of the development process.
+However, you may want to modify these to fit your own purposes.
+
+**It is unlikely that you will need to edit the pipeline code to configure a tool.**
+
+### Docker registries
```

Comment:
```suggestion
### Changing source of container registries
```

I think this section should go later, it is much less rare that someone do this..

Reply:
Moved